_tour/generic-classes.md

```diff
@@ -31,6 +31,8 @@ class Stack[A] {
 ```
 This implementation of a `Stack` class takes any type `A` as a parameter. This means the underlying list, `var elements: List[A] = Nil`, can only store elements of type `A`. The procedure `def push` only accepts objects of type `A` (note: `elements = x :: elements` reassigns `elements` to a new list created by prepending `x` to the current `elements`).
 
+`Nil` here is an empty `List` and is not to be confused with `Null`.
```

Comment:
`Nil` is a term, not a type, so I think the term `null` would make more sense here, `Null` is the type.